doc/source/dangerous_tips.rst

```diff
@@ -0,0 +1,72 @@
+Dangerous Tips
+++++++++++++++
+
+Only for experts in need for desperate ideas to increase the development process.
+
+Disclaimer / Warning
+--------------------
+
+If you see this page, it must be after multiple considerations whether it should or
+should not be published within OpenUpgrade official documentation.
+
+Why not publish it?
+
+Because **the content of this page might lead you to big unrecoverable mess in your
+data.**
+
+Why still publish it?
+
+Because you might be working on a very heavy migration, with a very large size database,
+and if you know what you are doing, this page will help you saving time during the
+development process of migration scripts.
+
+
+Dangerous tip: uninstall modules which are not yet covered
+----------------------------------------------------------
+
+For some of the uncovered modules, you might want to uninstall
+them from your database and re-install them later on after the migration.
+
+This might reduce the workload of the migration and can be applicable
+for "ux" modules, like ``web_widget_image_download``.
+
+But, if you uninstall the wrong modules, **you might end up in losing data.**
+
+To uninstall module, you can execute:
+
+::
+
+   UPDATE ir_module_module
+   SET state = 'to remove'
+   WHERE
+       state IN ('to upgrade', 'to install')
+       AND name in ('module_to_uninstall_1', 'module_to_uninstall_2')
+   RETURNING name
+
+Then run Odoo with ``-u dummy --stop-after-init`` to uninstall the modules
+that could not be upgraded yet.
+
+That way, you get a partially migrated database that you can login to and
+perform UI testing.
+
+
+Dangerous tip: ``fsync = off`` in your postgresql.conf
```

Comment:
Still too dangerous for my taste, sorry. Don't mention this at all, please.

Reply:
Yes, I agree with Stefan.